src/coreclr/tools/aot/ILCompiler.DependencyAnalysisFramework/ILCompiler.DependencyAnalysisFramework.csproj

```diff
@@ -15,11 +15,6 @@
     <GenerateDependencyFile>false</GenerateDependencyFile>
     <Configurations>Debug;Release;Checked</Configurations>
   </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="System.Collections.Immutable">
-      <Version>$(SystemCollectionsImmutableVersion)</Version>
```

Comment:
This was the last usage of `SystemCollectionsImmutableVersion`, we can delete it from https://github.com/dotnet/runtime/blob/8f7761af3763d0b84acefbb117ed6becd83061d1/eng/Versions.props#L97